www/base/src/styles/styles.less

```diff
@@ -1,8 +1,11 @@
 // Import bootstrap and font awesome stylesheets
-@import "../../libs/bootstrap/less/bootstrap.less";
-@import "../../libs/font-awesome/less/font-awesome.less";
-@import (inline) "../../libs/guanlecoja-ui/styles.css";
-@fa-font-path: "./fonts";
+@import "../app/**/*.less";
+@import "./animations.less";
+@import "./colors.less";
+@import "./mobile.less";
+@import "~bootstrap/less/bootstrap.less";
```

Comment:
I'd expect bootstrap to be first. in css, order is important, and can lead with subtle issues

Reply:
Agreed.

Reply:
Fixed now.